app/_src/gateway/get-started/services-and-routes.md

```diff
@@ -184,7 +184,7 @@ complete that before proceeding.
 The [Admin API documentation](/gateway/latest/admin-api/#update-service) provides
 the full service update specification. 
 
-You can also view the configuration for your services in the Kong Manager UI by navigating to the following URL in your browser: [http://localhost:8002/default/services](http://localhost:8002/default/services)
+You can also view the configuration for your services in the Kong Manager UI by navigating to the following URL in your browser: [http://localhost:8002/services](http://localhost:8002/services)
```

Comment:
```suggestion
You can also view the configuration for your services in the Kong Manager UI by navigating to the following URL in your browser: 
* Kong Manager OSS: [http://localhost:8002/services](http://localhost:8002/services) 
* Kong Manager Enterprise: [http://localhost:8002/default/services](http://localhost:8002/default/services), where `default` is the workspace name.
```
The `default` URL does work, but only in the Enterprise Kong Manager. In the OSS one, there's no workspaces.